go.mod

```diff
@@ -13,6 +13,7 @@ exclude (
 
 require (
 	github.com/google/go-cmp v0.5.0
+	github.com/hashicorp/go-multierror v1.1.0
```

Comment:
go mod tidy?

Reply:
Negative; I did indeed try this. `go mod why` details it's because of the helper package:
```
# github.com/hashicorp/go-multierror
github.com/hashicorp/terraform-provider-nomad/nomad/helper
github.com/hashicorp/go-multierror
```

Reply:
I probably forgot to commit this when I created the StateWriter helper..